joblib/test/test_parallel.py

```diff
@@ -768,3 +769,42 @@ def generate_arrays(n):
         delayed(check_memmap)(a) for a in generate_arrays(100))
     for result, expected in zip(results, generate_arrays(len(results))):
         np.testing.assert_array_equal(expected, result)
+
+
+@with_multiprocessing
+def test_nested_parallel_warnings():
+    # The warnings happen in child processes so
+    # warnings.catch_warnings can not be used for this tests that's
+    # why we use check_subprocess_call instead
+    if posix is None:
+        # This test pass only when fork is the process start method
+        raise nose.SkipTest('Not a POSIX platform')
+
+    template_code = """
+import sys
+
+from joblib import Parallel, delayed
+
+
+def func():
+    return 42
+
+
+def parallel_func():
+    res =  Parallel(n_jobs={inner_n_jobs})(delayed(func)() for _ in range(3))
+    # Needed otherwise some warnings may be missed
```

Comment:
not super critical but I think this comment is useless or there's something missing after.